ca/certificates_test.go

```diff
@@ -954,6 +938,12 @@ func TestGetRemoteSignedCertificateConnectionErrors(t *testing.T) {
 }
 
 func TestNewRootCA(t *testing.T) {
+	// even if root encryption passphrases are set, they are not used
```

Comment:
what does this mean? can you add some more comments explaining what this accomplishes and tests?